postStart.sh

```diff
@@ -46,5 +46,10 @@ if [[ -d "/root/repositoryscripts/src" && -d "/root/repositoryscripts/vars" ]];
 
 fi
 
-chmod -R 700 /root/.ssh
-chmod -R 700 /home/jenkins/.ssh-git
+if [ -f "/root/.ssh" ]; then
```

Comment:
Shouldn't this be `-d` (directory) rather than `-f` (regular file, not directory?)